cymetric/metrics.py

```diff
@@ -410,6 +410,26 @@ def annual_electricity_generated_by_agent(series):
 
 del _egdeps, _egschema
 
+
+# Mass of SNF+HLW disposed
+dependencies = [
+	('Mass', (Col1), 'Value1')
+	('Energy', (Col2), 'Value2')
+	]
+
+schema = [('SimId', ts.INT), ('Mass', ts.INT), ('Energy', ts.DOUBLE)]
+
+@metric(name='MassOfSNF+HLWdisposedPerEnergyGenerated', depends=dependencies, schema=schema)
```

Comment:
Metric names shouldn't have special characters. Please replace the `+` with something else.